src/dreyfus_index_updater.erl

```diff
@@ -64,7 +73,7 @@ update(IndexPid, Index) ->
         couch_db:close(Db)
     end.
 
-load_docs(FDI, _, {I, IndexPid, Db, Proc, Total, LastCommitTime}=Acc) ->
+load_docs(FDI, {I, IndexPid, Db, Proc, Total, LastCommitTime}=Acc) ->
```

Comment:
We used a new function `couch_db:fold_changes` here, and this function need a function with 2 arguments.